.gitignore

```diff
@@ -58,3 +58,6 @@ target/
 
 # Pycharm
 .idea/
+
+# Whylog
+.whylog_test
```

Comment:
You should never see this directory. Mea culpa. A forgot write handler which delete this directory if ConfigFactory tests fail. I added it in #101. If you got this directory it means something gone wrong. .gitignore can hide this.